src/log4net/log4net.csproj

```diff
@@ -66,6 +37,20 @@
     <DocumentationFile>..\..\build\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
+  <PropertyGroup Label="NuGet generation">
+    <Authors>The Apache Software Foundation</Authors>
+    <Copyright>Copyright 2004-2024 The Apache Software Foundation</Copyright>
+    <DevelopmentDependency>false</DevelopmentDependency>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageProjectUrl>https://logging.apache.org/log4net/</PackageProjectUrl>
+    <PackageIcon>package-icon.png</PackageIcon>
+    <PackageTags>logging log tracing logfiles</PackageTags>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/apache/logging-log4net</RepositoryUrl>
+    <Owners>Apache Logging Project, Jiří Činčura</Owners>
```

Comment:
I think this should be changed to "Apache Logging Project" - Jiří Činčura is github user cincuranet and I can neither find github issues/prs from that user nor can I find commits with that name or username. The project is owned by the Apache Logging Project, afaik.

Reply:
Jiří Činčura was the first one to push log4net to nuget.org
I've changed it.